CoDebt/core/managers.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import BaseUserManager
+
+class CustomUserManager(BaseUserManager):
+    """
+    Custom user model manager where email is the unique identifiers
+    for authentication instead of usernames.
+    """
+    def create_user(self, email, password, **extra_fields):
```

Comment:
This is the class authentication right?
I don't see where we implemented schools uploading a verification document before they are authorized as school administrators